### ISSUE TITLE
README: use CMAKE_ARGS instead of -C cmake.args

### ISSUE THREAD

**Issue by alinaryan** [instructlab/instructlab#1873]
In the README at the [install with Nvidia CUDA step](https://github.com/instructlab/instructlab?tab=readme-ov-file#install-with-nvidia-cuda), we need to replace -C cmake.args with CMAKE_ARGS because -C is not a valid option

**Comment by bjhargrave** [comment]
You may need to upgrade pip to the latest version.

**Comment by tiran** [comment]
I agree with @bjhargrave analysis. You are most likely running an older pip. Upstream only supports latest version. A virtual env with `python3 -m venv --upgrade-deps ...` installs the latest pip.

However it may be still a good idea to get rid of all `-C cmake.args`. It sometimes does not work and I don't know why. See #1829

**Comment by cdoern** [comment]
I think its fair to add a doc that says `if -C fails either specify --upgrade-deps or use CMAKE_ARGS`. Just because this works on the most recent version of pip doesn't necessarily mean this isn't something we should document for less experienced users.

**Comment by ktam3** [comment]
@alinaryan - is there a PR for this?

**Comment by nathan-weinberg** [comment]
@alinaryan is this a dup of https://github.com/instructlab/instructlab/issues/1354?

**Comment by alinaryan** [comment]
@nathan-weinberg Closing this, this issue was resolved by #1354
